incubator/quarkus/image/Dockerfile-stack

```diff
@@ -12,6 +12,7 @@ RUN mvn -B install dependency:go-offline -DskipTests
 
 ENV APPSODY_MOUNTS=".:/project/user-app"
 ENV APPSODY_DEPS="/home/quarkus/.m2"
+ENV APPSODY_PREP="mkdir /project/user-app/target"
```

Comment:
Is this the right place for doing this? If we call an `appsody run/test/debug` more than once, the command will fail because of:
```
[Container] Running APPSODY_PREP command: mkdir /project/user-app/target
[Container] mkdir: cannot create directory '/project/user-app/target': File exists
[Container] [Error] FATAL error APPSODY_PREP command received an error.  The controller is exiting: exit status 1
```
Could this dir be created in `.appsody-init.sh`?

Reply:
`mkdir -p` does fix it, thanks. There was some discussion about using `.appsody-init.sh`. But I think the agreement was to do it this way now as it was simpler and we expect a better solution to come soon.